### ISSUE TITLE
Import error

### ISSUE THREAD

**Issue by jkosinski** [KosinskiLab/colabseg#3]
On an M2 MacBook when running the following in the colabseg_demo_notebook (but after installing with python 3.9 as python 3.8 was not available):
```
from colabseg.tensorvoting_wrapper import *
from colabseg.segmentation_gui import *
 ```

I am getting:
```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[1], line 1
----> 1 from colabseg.tensorvoting_wrapper import *
      2 from colabseg.segmentation_gui import *

File ~/devel/colabseg/colabseg/__init__.py:9
      1 #
      2 # ColabSeg - Interactive segmentation GUI
      3 #
   (...)
      6 # from tensorvoting_wrapper import *
      7 # from segmentation_gui import *
----> 9 from .new_gui_functions import ColabSegData
     10 from .image_io import ImageIO

File ~/devel/colabseg/colabseg/new_gui_functions.py:11
      9 import h5py
     10 import numpy as np
---> 11 import open3d as o3d
     12 import scipy.spatial as spatial
     13 from scipy import interpolate

File ~/anaconda3/envs/ColabSeg/lib/python3.9/site-packages/open3d/__init__.py:93
     87         warnings.warn(
     88             "Open3D was built with CUDA support, but Open3D CUDA Python "
     89             "binding library not found! Falling back to the CPU Python "
     90             "binding library.", ImportWarning)
     92 if __DEVICE_API__ == "cpu":
---> 93     from open3d.cpu.pybind import (core, camera, data, geometry, io, pipelines,
     94                                    utility, t)
     95     from open3d.cpu import pybind
     98 def _insert_pybind_names(skip_names=()):

ImportError: dlopen(/Users/kosinski/anaconda3/envs/ColabSeg/lib/python3.9/site-packages/open3d/cpu/pybind.cpython-39-darwin.so, 0x0002): Library not loaded: /opt/homebrew/opt/libomp/lib/libomp.dylib
  Referenced from: <6C0FB50D-E600-3D2F-AF11-C0935C1FBD7C> /Users/kosinski/anaconda3/envs/ColabSeg/lib/python3.9/site-packages/open3d/cpu/pybind.cpython-39-darwin.so
  Reason: tried: '/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/usr/local/lib/libomp.dylib' (no such file), '/usr/lib/libomp.dylib' (no such file, not in dyld cache)
```

**Comment by jkosinski** [comment]
Same error with python3.8

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[1], line 1
----> 1 from colabseg.tensorvoting_wrapper import *
      2 from colabseg.segmentation_gui import *

File ~/devel/colabseg/colabseg/__init__.py:9
      1 #
      2 # ColabSeg - Interactive segmentation GUI
      3 #
   (...)
      6 # from tensorvoting_wrapper import *
      7 # from segmentation_gui import *
----> 9 from .new_gui_functions import ColabSegData
     10 from .image_io import ImageIO

File ~/devel/colabseg/colabseg/new_gui_functions.py:11
      9 import h5py
     10 import numpy as np
---> 11 import open3d as o3d
     12 import scipy.spatial as spatial
     13 from scipy import interpolate

File ~/anaconda3/envs/ColabSeg/lib/python3.8/site-packages/open3d/__init__.py:93
     87         warnings.warn(
     88             "Open3D was built with CUDA support, but Open3D CUDA Python "
     89             "binding library not found! Falling back to the CPU Python "
     90             "binding library.", ImportWarning)
     92 if __DEVICE_API__ == "cpu":
---> 93     from open3d.cpu.pybind import (core, camera, data, geometry, io, pipelines,
     94                                    utility, t)
     95     from open3d.cpu import pybind
     98 def _insert_pybind_names(skip_names=()):

ImportError: dlopen(/Users/kosinski/anaconda3/envs/ColabSeg/lib/python3.8/site-packages/open3d/cpu/pybind.cpython-38-darwin.so, 0x0002): Library not loaded: /opt/homebrew/opt/libomp/lib/libomp.dylib
  Referenced from: <A2C68595-9594-3758-BEFB-86BA2C701093> /Users/kosinski/anaconda3/envs/ColabSeg/lib/python3.8/site-packages/open3d/cpu/pybind.cpython-38-darwin.so
  Reason: tried: '/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/opt/homebrew/opt/libomp/lib/libomp.dylib' (no such file), '/usr/local/lib/libomp.dylib' (no such file), '/usr/lib/libomp.dylib' (no such file, not in dyld cache)
```

**Comment by maurerv** [comment]
Fixed in 54be8648377c2c2b522a65ffe9efc09cfa240dd3
